lib/plugins/create/templates/aws-csharp/build.ps1

```diff
@@ -0,0 +1,19 @@
+# from http://stackoverflow.com/questions/1153126/how-to-create-a-zip-archive-with-powershell#answer-13302548
+function ZipFiles( $zipfilename, $sourcedir )
+{
+   Add-Type -Assembly System.IO.Compression.FileSystem
+   $compressionLevel = [System.IO.Compression.CompressionLevel]::Optimal
+   [System.IO.Compression.ZipFile]::CreateFromDirectory($sourcedir,
+        $zipfilename, $compressionLevel, $false)
+}
+
+dotnet restore
+dotnet publish -c release
+if ($LASTEXITCODE -ne 0) { return }
+
+$publishDirectory = "bin/Release/netcoreapp1.0/publish"
```

Comment:
Recently there was PR renaming "Release" to "release". I'm not C# expert. Which version is proper?

https://github.com/serverless/serverless/pull/3026